src/devtools/useAtomsDevtools.ts

```diff
@@ -52,11 +52,16 @@ const getDevtoolsState = (atomsSnapshot: AtomsSnapshot) => {
   }
 }
 
-export function useAtomsDevtools(name: string, scope?: Scope) {
-  const ScopeContext = getScopeContext(scope)
+interface DevtoolsOptions {
+  scope?: Scope
+  enabled?: boolean
+}
+
+export function useAtomsDevtools(name: string, options?: DevtoolsOptions) {
+  const ScopeContext = getScopeContext(options?.scope)
   const { s: store, w: versionedWrite } = useContext(ScopeContext)
 
-  if (!store[DEV_SUBSCRIBE_STATE]) {
+  if (!store[DEV_SUBSCRIBE_STATE] && options?.enabled && __DEV__) {
```

Comment:
I don't think we need this.
Ohhh! we can't enable this in PRD anyway! That was my confusion...

So, it should be something like this. Open for suggestions.
```js
if (enabled === false) {
  return
}
if (!store[DEV_SUBSCRIBE_STATE]) {
  throw new Error('useAtomsDevtools can only be used in dev mode.')
}
```

Reply:
This makes problems for us! make tests failing!